app/views/cash_account_declaration_details_search_no_result.scala.html

```diff
@@ -0,0 +1,91 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+       main_template: Layout,
+       h2: components.h2,
+       h1Inner: components.h1Inner,
+       h2Inner: components.h2Inner,
+       p: components.p,
+       ul: components.unorderedList,
+       link: components.link,
+       newTabLink: components.newTabLink
+)
+
+@(
+    pageNumber: Option[Int],
+    accountNumber: String,
+    searchInput: String
+)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(
+  pageTitle = Some(messages("cf.cash-account.detail.title")),
+  backLink = Some(routes.CashAccountV2Controller.showAccountDetails(page = pageNumber).url),
+  fullWidth = false
+) {
+
+  @h2Inner(
+     msg = "cf.cash-account.detail.account",
+     innerMsg = accountNumber,
+     id = Some("account-number"),
+     classes = "govuk-caption-xl"
+  )
+
+  @h1Inner(
+     msg = "cf.cash-account.detail.declaration.search-title",
+     innerMsg = searchInput,
+     id = Some("search-result-heading")
+  )
+
+  @p(
+     message = messages("cf.cash-account.detail.declaration.search-no-results-guidance-not-returned-any-results", searchInput),
+     id = Some("search-result-guidance-not-returned-any-results")
+  )
+
+  @p(
+     message = messages("cf.cash-account.detail.declaration.search-no-results-guidance-because-you-entered"),
+     id = Some("search-result-guidance-because-you-entered")
+  )
+
+  @ul(
+     id = Some("invalid-inputs-guidance-list"),
+     liMsgList = List(
+                    messages("cf.cash-account.detail.declaration.search-no-results-invalid-input-mrn"),
+                    messages("cf.cash-account.detail.declaration.search-no-results-invalid-input-payment-amount"),
+                    messages("cf.cash-account.detail.declaration.search-no-results-invalid-input-pound-sign")
+     )
+  )
+
+  @link(
+     linkMessageKey = "site.search.again",
+     location = routes.CashAccountV2Controller.showAccountDetails(pageNumber).url,
+     pId = Some("search-again-link")
+  )
+
+  @h2(
+      msg = "cf.cash-account.transactions.request.support.heading",
+      id = Some("cash-account-help-and-support-guidance-header"),
+      classes = "govuk-heading-m govuk-!-margin-top-9",
+  )
+
+  @newTabLink(
+      linkMessage = "cf.cash-account.help-and-support.link.text",
+      href = appConfig.cashAccountForCdsDeclarationsUrl,
+      preLinkMessage = Some("cf.cash-account.help-and-support.link.text.pre"),
+      postLinkMessage = Some("cf.cash-account.help-and-support.link.text.post")
+  )
+
```

Comment:
unneeded space

Reply:
It is on purpose to make it readable by differentiating between ) and } (to show the end for newTabLink component)